data-prepper-plugins/otel-proto-common/src/test/java/org/opensearch/dataprepper/plugins/otel/codec/OTelLogsProtoBufDecoderTest.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.otel.codec;
+
+import java.io.InputStream;
+import java.time.Instant;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+import org.opensearch.dataprepper.model.log.OpenTelemetryLog;
+import org.opensearch.dataprepper.model.event.Event;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class OTelLogsProtoBufDecoderTest {
+    private static final String TEST_REQUEST_LOGS_FILE = "test-otel-log.protobuf";
+    private static final String TEST_REQUEST_MULTI_LOGS_FILE = "test-otel-multi-log.protobuf";
+    private int count;
+    
+    public OTelLogsProtoBufDecoder createObjectUnderTest(boolean lengthPrefixedEncoding) {
+        return new OTelLogsProtoBufDecoder(lengthPrefixedEncoding);
+    }
+
+    private void validateLog(OpenTelemetryLog logRecord, final int severityNumber, final String time, final String spanId) {
```

Comment:
Would `assertLog` be a better name?

Reply:
We have been using this name in other places as well. I think both names are good.

Reply:
I agree that `assertLog` would be clearer because that is exactly what we are doing here. We should use "validate" for things like input validation and configuration validation.